azkaban-common/src/test/java/azkaban/jobExecutor/JavaProcessJobTest.java

```diff
@@ -108,13 +108,15 @@ public void setUp() throws IOException {
     props.put(AbstractProcessJob.WORKING_DIR, workingDir.getCanonicalPath());
     props.put("type", "java");
     props.put("fullPath", ".");
-    
+
     props.put(CommonJobProperties.PROJECT_NAME, "test_project");
     props.put(CommonJobProperties.FLOW_ID, "test_flow");
     props.put(CommonJobProperties.JOB_ID, "test_job");
     props.put(CommonJobProperties.EXEC_ID, "123");
     props.put(CommonJobProperties.SUBMIT_USER, "test_user");
-    
+
+    //For local test case to pass, we need to execute the jobs as Azkaban instead of effectiveUser.
```

Comment:
Consider adding why in the comment. e.g.

The execute_as_user binary requires special permission. It's not convenient to set up in a unit test that is self contained.